### PR TITLE
Add pymorton

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
A few notes about this package:
- Unfortunately, it does not seem to be actively maintained anymore. The last commit is from 4 years ago.
- The latest Pypi release is v1.0.5 but does not have source files
- The latest GitHub release is v1.0.5 - I use this as the source file
- The latest release version on main is v1.0.7. I use https://github.com/trevorprater/pymorton/compare/1.0.5...master.diff to get the diff
- I fixed a bug and created a PR: https://github.com/trevorprater/pymorton/pull/6
- Other community members see a need for the package for conda-forge: https://github.com/trevorprater/pymorton/issues/4

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-python this is ready for review :). Many thanks!

File: recipes/pymorton/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trevorprater/{{ name }}/archive/refs/tags/1.0.5.tar.gz
+  sha256: f22ebe8c1f69ef4001c59f277dbe5403cb18f6bd3004ef19ac0e5fbdf5f865f6
+  patches:
+    - v1.0.7.patch
+    - fix-64-bit.patch
```
Comment (ocefpaf) [inline recipes/pymorton/meta.yaml:13]: Can you add a comment on why these patches are needed and/or if they are upstream?

Comment (Tobias-Fischer) [reply recipes/pymorton/meta.yaml]: Thanks for the review, @ocefpaf, all done :)

**Comment by Tobias-Fischer** [comment]
Thanks for merging!

**Pull request merged**